src/app/legacy/providers/redeem-provider.tsx

```diff
@@ -97,42 +98,25 @@ export function RedeemProvider(props: { children: any }) {
       type: QuoteType.issuance,
       isAvailable: true,
       quote: null,
+      legacy: null,
       error: null,
     })
   }
 
-  useEffect(() => {
-    const fetchOutputToken = async (inputToken: Token) => {
-      if (!publicClient) return
-      const outputToken = await getOutputToken(
-        inputToken.address!,
-        publicClient,
-      )
-      if (!outputToken) return
-      setOutputToken(outputToken)
-    }
-    fetchOutputToken(inputToken)
-  }, [inputToken, publicClient])
-
   useEffect(() => {
     const fetchQuote = async () => {
```

Comment:
this could probably be replaced by `useQuery`, sparing the setFetchingQuote(true) + the other advantages